tests/director/tests/application/application_templates_api_test.go

```diff
@@ -102,13 +106,187 @@ func TestCreateApplicationTemplate(t *testing.T) {
 		t.Log("Create application template")
 		err = testctx.Tc.RunOperationWithCustomTenant(ctx, certSecuredGraphQLClient, tenantID, createApplicationTemplateRequest, &output)
 		defer fixtures.CleanupApplicationTemplate(t, ctx, certSecuredGraphQLClient, tenantID, output)
+		require.NoError(t, err)
 
 		// THEN
 		require.Equal(t, output.Labels[conf.ApplicationTemplateProductLabel], productLabelValue)
-		require.NoError(t, err)
 		require.NotEmpty(t, output.ID)
 	})
 
+	t.Run("Success for regional template with product label created with certificate", func(t *testing.T) {
+		// GIVEN
+		ctx := context.Background()
+
+		productLabelValue := []interface{}{"productLabelValue"}
+		appTemplateName1 := fixtures.CreateAppTemplateName("app-template-name-product-1")
+		appTemplateInput1 := fixtures.FixRegionalApplicationTemplate(appTemplateName1, region1, regionPlaceholderJSONPath1)
+		appTemplateInput1.Labels[conf.ApplicationTemplateProductLabel] = productLabelValue
+		appTemplate1, err := testctx.Tc.Graphqlizer.ApplicationTemplateInputToGQL(appTemplateInput1)
+		require.NoError(t, err)
+
+		createApplicationTemplateRequest1 := fixtures.FixCreateApplicationTemplateRequest(appTemplate1)
+		output1 := graphql.ApplicationTemplate{}
+
+		appTemplateName2 := fixtures.CreateAppTemplateName("app-template-name-product-2")
```

Comment:
Create application templates with the same name but different regions, so that the regional functionality can be tested. The comment is regarding the newly introduced tests.